features/step_definitions/logging.rb

```diff
@@ -705,3 +705,37 @@
     end
   end
 end
+
+# Delete the fluentd buffer files in nodes. the fluentd must be deleted before running this step.
+Given /^logging collector bufferfiles are cleared on all nodes$/ do
+  ensure_admin_tagged
+  serviceaccount="collector-clean"
+  ds_name="collector-clean"
+  user(0)
+  step %Q/I have a project/
```

Comment:
I don't see any steps to delete the project, will the project be deleted when the step/scenario finished?

Reply:
Yes.  the cronjob will be running under one existing project if there is.

Reply:
I suggest you to remove the step to create the project and suggest you to create the project before you call the step `I clear all logging collector buffer files all nodes`  in the scenario.